xarray/core/indexing.py

```diff
@@ -1513,23 +1514,29 @@ def __repr__(self) -> str:
             )
             return f"{type(self).__name__}{props}"
 
-    def _repr_inline_(self, max_width) -> str:
-        # special implementation to speed-up the repr for big multi-indexes
+    def _get_array_subset(self, size: int) -> np.ndarray:
+        # used to speed-up the repr for big multi-indexes
+        if self.size > 200 and size < self.size:
```

Comment:
```suggestion
        if self.size > MAX_MULTI_INDEX_REPR_SIZE and size < self.size:
```

Shall we add a constant for this magic number?

Reply:
Agreed. Should we make it an option? This magic number is also used here:

https://github.com/pydata/xarray/blob/f8bae5974ee2c3f67346298da12621af4cae8cf8/xarray/core/formatting.py#L523

Reply:
Sure,`DISPLAY_MAX_ITEMS`?